README.md

```diff
@@ -30,6 +30,7 @@ _Note that smart contracts decentralized applications also function as libraries
   - [Funding programs](#funding-programs)
   - [Content](#content)
   - [Tutorials](#tutorials)
+  - [Zero-knowledge proofs](#zero-knowledge-proofs)
```

Comment:
Should be a subcategory of the 'projects' section

Reply:
done